test/itemlistformaction.cpp

```diff
@@ -133,6 +133,40 @@ TEST_CASE(
 	REQUIRE(feed->unread_item_count() == 0);
 }
 
+TEST_CASE(
+	"OP_OPENBROWSER_AND_MARK does not mark read when browser fails",
+	"[ItemListFormAction]")
+{
+	Controller c;
+	newsboat::View v(&c);
+	TestHelpers::TempFile browserfile;
+
+	std::string test_url = "http://test_url";
+	std::string line;
```

Comment:
This can be removed as well, for the same reason as `browserfile` above.

Reply:
@mjsir911, you marked this comment as resolved, but the variable is still there?